Halo-prvsnr/ansible/build_and_load_images/build_images/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.10
```

Comment:
This repo hosts code written for POCs and if the poc is specific to the technology then it should go to corresponding technology folder. Please change base directory to ansible.

Reply:
Ack.
change the base dir from Halo-prvsnr/ansible to ansible